server/services/discord/actions/on_reaction_add.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+const client = require('../connect')
+
+/**
+ * @brief Triggered when a user reacts to a message.
+ * @param {*} reaction Reaction object
+ * @param {*} user User that reacted to the message
+ */
+client.on('messageReactionAdd', (reaction, user) => {
```

Comment:
Trigered in every discord server and MP ?

Reply:
Yes that's it. So when we will use it, we will need to check the server and channels/messages IDs if needed